co-operative.md

```diff
@@ -0,0 +1,50 @@
+# Hypha Worker Co-operative
+
+## About Us 
+
+Hypha Worker Co-operative Inc. is a worker co-operative incorporated on August 
+1, 2019 in Ontario, Canada (Ontario Corporation No. 5019866).
+
+Hypha is incorporated as a **non-share capital** worker co-operative under the 
+[*Ontario Co-operative Corporations Act, 1990*][coop-act]. This entails specific 
```

Comment:
```suggestion
[_Ontario Co-operative Corporations Act, 1990_][coop-act]. This entails specific 
```

I've always seen this as the informal visual distinction, and thought we were following this up til now? nbd though

Reply:
Should we just straight up clarify here (or nearby) that:

> As per 2017's [_Cutting Unnecessary Red Tape Act_](https://www.ontario.ca/laws/statute/s17020), Hypha doesn't fall under the current [_Corporations Act_](https://www.ontario.ca/laws/statute/90c38), nor the upcoming [_Not-for-Profit Corporations Act_](https://www.ontario.ca/laws/statute/10n15).

Reply:
I've had issues with `_` underscores correctly rendering as italicized across line breaks in some markdown generation lately, so I've switched to using `*` a single asterisk. 

I'm okay enforcing the visual distinction, this is just to ensure rendering correctly :woman_shrugging:  and it's invaded my typing

Reply:
I have no issue sticking with `*` if that's the case.

Reply:
ok cool. not know renderer or example, hard to diagnose or assess, but sounds good.